Client/NoOpRunner.Client/MainWindow.xaml.cs

```diff
@@ -43,9 +50,25 @@ private void TriggerRender()
 
             Game.FireLoop();
 
-            GameWindowRenderer.Render(Game.GameWindow, canvas);
+            GameWindowRenderer.RenderPlayer(Game.Player, player_window, Game.GameWindow);
+
+            GameWindowRenderer.Render(Game.GameWindow, canvas);//TODO: don't render if platforms don't move, event on window resize to change platforms W, H 
+        }
+
+        private void SetUpBackground()
+        {
+            game_window.Background = new ImageBrush(InitBitmapImage(SpritesUriHandler.GetBackground()));//SHIT
         }
 
+        private static BitmapImage InitBitmapImage(Uri uri)
+        {
+            var bitmapImage = new BitmapImage();
+            bitmapImage.BeginInit();
+            bitmapImage.UriSource = uri;
+            bitmapImage.EndInit();
+
+            return bitmapImage;
+        }
```

Comment:
Please remove the glue from your enter button and leave empty lines between methods.

Reply:
I got big ENTER, it's US standard, need to adapt